examples/terraform-azure-aks-example/variables.tf

```diff
@@ -0,0 +1,38 @@
+
+
```

Comment:
NIT: We like to avoid unnecessary whitespace.

Reply:
Also, we usually provide a description for all variables so that it is clear what should be put in here. E.g I do not know at a glance if `client_secret` refers to secret key for k8s, or for Azure.

Reply:
Done